iotedgedev/connectionstring.py

```diff
@@ -13,8 +16,16 @@ def __init__(self, value):
 
             if self.data:
                 self.HostName = self["hostname"]
-                if self.HostName:
+                if self.HostName and "." in self.HostName:
```

Comment:
Alternatively you could expand "Host"  to a class, so you can have: Host.Name, Host.NameHashed, Host.Suffix

Reply:
Thanks. Updated.